csm/core/agent/api.py

```diff
@@ -292,10 +292,9 @@ async def session_middleware(request, handler):
         is_public = await CsmRestApi._is_public(request)
         is_hybrid = await CsmRestApi._is_hybrid(request)
         if is_hybrid:
+            is_public = True
             conf_key = CsmRestApi._retrieve_config(request)
-            if conf_key == "disable" or conf_key == "Disable":
-                is_public = True
-            else:
+            if conf_key == "enable" or conf_key == "Enable":
```

Comment:
enable/Enable keep this in const file. I

Reply:
Added const as per comment